src/InputWithLabel.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useRef, useEffect } from "react";
+
+const InputWithLabel = ({ id, children, value, onChange }) => {
+  const inputRef = useRef();
+
+  useEffect(() => {
+    if (inputRef.current) {
+      inputRef.current.focus();
+    }
+  }, []);
```

Comment:
nit: the assignment is asking to not have a dependency list, which is different from having an empty dependency list. difference
- empty array: runs once when the component initially renders (mount)
- no array: runs every time the component updates/re-renders